trillian/ctfe/ct_server/main.go

```diff
@@ -79,9 +90,6 @@ func main() {
 		metricsAt = *httpEndpoint
 	}
 
-	// TODO(Martin2112): Support TLS and other stuff for RPC client and http server, this is just to
```

Comment:
This TODO seems to be evaporating without being completely done.

Reply:
We haven't seen any requirements for CTFE to terminate HTTPS yet. Probably better to address it when it arises rather than promise to something we don't really plan to do.

Reply:
OK, fair enough